ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml

```diff
@@ -10,6 +10,13 @@ base_images:
 build_root:
   from_repository: true
 images:
+- dockerfile_path: Dockerfile
```

Comment:
this is path is for https://github.com/openshift/cluster-node-tuning-operator/blob/master/Dockerfile ?